ios/Podfile

```diff
@@ -39,9 +39,7 @@ target 'Backpack Native' do
   # RN Bridging Pods
   pod 'Backpack', '~> 26.0'
 
-  pod 'react-native-bpk-component-calendar', path: '../packages/react-native-bpk-component-calendar', :modular_headers => false
-  pod 'react-native-bpk-component-dialog', path: '../packages/react-native-bpk-component-dialog', :modular_headers => false
-  pod 'react-native-bpk-component-rating', path: '../packages/react-native-bpk-component-rating', :modular_headers => false
+  pod 'backpack-react-native', path: '../lib/src/ios/backpack-react-native', :modular_headers => false
```

Comment:
Can we call this pod `BackpackReactNative` instead? The dash convention isn't used for pod names